target/linux/brcmiproc/base-files/etc/inittab

```diff
@@ -0,0 +1,6 @@
+::sysinit:/etc/init.d/rcS S boot
+::shutdown:/etc/init.d/rcS K shutdown
+ttyAMA0::askfirst:/usr/libexec/login.sh
+ttyS0::askfirst:/usr/libexec/login.sh
+ttyS1::respawn:/usr/libexec/login.sh
+hvc0::askfirst:/usr/libexec/login.sh
```

Comment:
I would move the whole base-files folder to the subtarget already, so 

target/linux/brcmiproc/base-files/etc/inittab

to

target/linux/brcmiproc/bcm58800/base-files/etc/inittab

and so on.

This will keep them for the subtarget only, so when another subtarget is added, it starts off "clean".

Reply:
Moved base-files folder to subtarget